### PR TITLE
[MIG] 17.0 l10n_fi_banks 

### PR DESCRIPTION
**Pull request by jarmokortetjarvi**
Migrate from version 14.0
Added BIGKFIH1

Actual changes
https://github.com/jarmokortetjarvi/l10n-finland/compare/14.0...jarmokortetjarvi:l10n-finland:17.0-mig-l10n_fi_banks?expand=1

### CODE REVIEW

**Comment by aisopuro** [comment]
LGTM, but I cannot merge.

File: pandoc-3.1.11-1-amd64.deb
Comment (mlaitinen) [inline pandoc-3.1.11-1-amd64.deb:1]: This binary file doesn't seem to belong here

**Review by mlaitinen — CHANGES_REQUESTED** [review]
LGTM, but the binary file `pandoc-3.1.11-1-amd64.deb` should be removed before merging.

**Comment by jarmokortetjarvi** [comment]
Removed the binary file. Merging, as this was otherwise reviewed

**Pull request merged**